python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -49,14 +50,17 @@ def __init__(
         native_support_kwargs: dict | None = None,
         columns_names_capitalization: ColumnCapitalization = "original",
         enable_native_fallback: bool | None = LOAD_FILE_ENABLE_NATIVE_FALLBACK,
+        task_id: str | None = None,
         **kwargs,
     ) -> None:
+        task_id = task_id or get_unique_task_id("load_file")
         super().__init__(
+            task_id=task_id,
```

Comment:
@pankajkoti maybe we also need the task_id actually in `kwargs_with_datasets`? Did you test this?

Reply:
Or maybe only there? And it will be passed to the `super().__init__()` automatically.

Reply:
yes, tested @feluelle . We don't need it there :)